docs/weave-tutorial/01-overview.md

```diff
@@ -0,0 +1,16 @@
+---
+id: overview
+title: Tutorial Overview
+sidebar_label: Overview
+---
+
+First of all, to understand weave design philosophy and for the sake of programming well-designed software, I advise you to read this fine-grained article:
```

Comment:
"First of all, to understand weave design philosophy and for the sake of programming well-designed software, I advise you to read this fine-grained article:"

"First of all, to understand the design philosophy behind Weave and in the interest of good software design, a recommended read is:"